content/en/docs/contribute/style/style-guide.md

```diff
@@ -58,11 +58,11 @@ leads to an awkward construction.
 {{< table caption = "Do and Don't - API objects" >}}
 Do | Don't
 :--| :-----
-The Pod has two containers. | The pod has two containers. 
+The Pod has two containers. | The pod has two containers.
 The Deployment is responsible for ... | The Deployment object is responsible for ...
 A PodList is a list of Pods. | A Pod List is a list of pods.
-The two ContainerPorts ... | The two ContainerPort objects ... 
-The two ContainerStateTerminated objects ... | The two ContainerStateTerminateds ... 
```

Comment:
Note: before anyone comments on all the spaces in this PR – I think it's my editor (Atom). I'll look into correcting that :(

Reply:
@celestehorgan I think it's fine to clean up whitespace; the trick is to do that clean up in one commit and then make other changes in a second commit.

(I haven't looked at the preview yet, might have time to review in the future).